Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -249,22 +642,46 @@ The following rules need to be followed, in addition to the guidelines described
    Make sure sequences are tested for all modes of operation, and as applicable, for winter, shoulder
    and summer days.
 
+#. Perform code text and, if applicable, figure inspection to:
+      - ensure there are no remaining typos
+      - ensure no element `public` was added automatically after the `protected` code section
 
+#. Use full paths to blocks, that is paths that start with `Buildings.Controls...`
 
-.. _sec_val:
+#. Final steps for a pull requests:
+      - make sure that the latest base branch is merged into the development branch
+      - after creating a pull request observe that the files changed pertain to the 
+        issue being addressed with the development branch
 
-Validation
-----------
+#. For general rules on validation models see :numref:`sec_val`. Specifically for the 
+   sequence development ensure that:
+      - the instances of models being tested are listed at the top of the code text
+      - add a blank line at the end of each .mos script
```

Comment:
Why is a blank line needed? If it is needed, is it a bug in BuildingsPy? If it is needed for Dymola, it belongs to the description of how to implement unit tests. This needs to be about control sequences only. If we start mixing it, we will repeat ourself and won't be concise.

Reply:
I assume it was added due to some tests failing if there isn't a blank line at the end of a mos file.

Reply:
I'll move these two statements to the unit testing wiki page:

* Protected instances are not accessible through `mos` scripts. 
* A blank line at the end of each .mos script

What's confusing is that we have a Validation and Unit Tests section in this file too.

Reply:
Moved.